plugins/hypervisors/ovm3/src/main/java/com/cloud/hypervisor/ovm3/resources/Ovm3VirtualRoutingResource.java

```diff
@@ -38,7 +40,7 @@
     private final Logger logger = Logger
             .getLogger(Ovm3VirtualRoutingResource.class);
     private String domRCloudPath = "/opt/cloud/bin/";
-    private int vrTimeout = 600;
+    private Duration vrTimeout = Duration.standardSeconds(600L);
```

Comment:
Please consider extracting ``Duration.standardSeconds(600L)`` to constant to clarify the intent.